kedro/framework/cli/starters.py

```diff
@@ -299,7 +337,16 @@ def new(  # noqa: PLR0913
 
     _create_project(project_template, cookiecutter_args)
 
-    if prompts_required and not config_path and not starter_alias:
+    # End here if a starter was used
+    if starter_alias:
```

Comment:
@AhdraMeraliQB, thank you for the excellent work, the code is much clearer now! 
Also, to further streamline our approach, perhaps we could consider this modification:
```
if not starter_alias:
   print_selection_and_prompt_info()
```
By consolidating all printing operations into a single function, I believe it will make the overall flow in the main create_cli() function easier to understand."

Reply:
Done in [12ba18c](https://github.com/kedro-org/kedro/pull/3484/commits/12ba18cd82766ccc36cf0fecbe69ff081549ef0d) !